packages/types/types/ts3.6/index.d.ts

```diff
@@ -1,6 +1,6 @@
 import { Action } from 'redux'
 
-export type Saga<Args extends any[] = any[]> = (...args: Args) => Iterator<any>
+export type Saga<Args extends any[] = any[], TReturn = any> = (...args: Args) => Iterator<any, TReturn>
```

Comment:
Note for me (and other reviewers) - this change is unique to 3.6+ types because previous versions of the `Iterator` didn't accept more than one generic.